src/main/java/sparksoniq/jsoniq/compiler/parser/JsoniqLexer.java

```diff
@@ -1,18 +1,20 @@
-// Generated from ./src/main/java/sparksoniq/jsoniq/compiler/parser/Jsoniq.g4 by ANTLR 4.5.3
+// Generated from ./src/main/java/sparksoniq/jsoniq/compiler/parser/Jsoniq.g4 by ANTLR 4.7
```

Comment:
Not even sure we should include these files at all in the repository since they are generated.

Reply:
I am not sure when they got added. I'll open an issue to have them removed, it the test queue can still manage.